### PR TITLE
Add godmode nft minter tags

### PR DESCRIPTION
**Pull request by PoolPirate**
Matches query results from
https://app.flipsidecrypto.com/velocity/queries/54fd0b68-97c1-4a2f-83c7-627e46cfeece

Given that all NFTs are already minted this list is complete and will no longer change.

### CODE REVIEW

File: data/silver_godmode_nft_minters.csv [hunk 1]
```diff
@@ -0,0 +1,454 @@
+blockchain,creator,address,tag_name,tag_type,start_date,end_date
```
Comment (gronders) [inline data/silver_godmode_nft_minters.csv:1]: please change the name of your file to 'silver__godmode_nft_minters.csv) (there is a double underscore after silver)

File: data/silver_godmode_nft_minters.csv [hunk 2]
```diff
@@ -0,0 +1,454 @@
+blockchain,creator,address,tag_name,tag_type,start_date,end_date
+ethereum,playwo,0x14742057dfd3ce5fd42e37c1fd5f9aef82f33061,Godmode Minter,NFT,2022-06-08 14:34:28.000,null
```
Comment (gronders) [inline data/silver_godmode_nft_minters.csv:2]: in the file 'models/core/core__address_tags.sql' please past this at the bottom (we will update the docs to include this step):
SELECT
    blockchain,
    creator,
    address,
    tag_name,
    tag_type,
    start_date,
    end_date,
    '2022-09-06'::timestamp AS tag_created_at
FROM
    {{ ref('silver__godmode_nft_minters') }}

**Review by gronders — CHANGES_REQUESTED** [review]
added 2 inline comments to fix, then we can merge into data :)

**Review by PoolPirate — COMMENTED** [review]
Done

**Review by gronders — APPROVED** [review]

**Pull request merged**